testing/watchexec/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Chloe Kudryavtsev <toast@toastin.space>
+# Maintainer: Chloe Kudryavtsev <toast@toastin.space>
+pkgname=watchexec
+pkgver=1.10.2
+pkgrel=0
+pkgdesc='Executes commands in response to file modifications'
```

Comment:
should use " " not ' ' for quoting variables

Reply:
Is there a standard saying this?
Generally, in shell, `''` quotes mean "no substitution", while `""` mean that substitution is allowed.
As such, if anything, constants should prefer using `''` (unless there's a spec/standard/policy saying otherwise, which is why I ask).

Reply:
No, it is just what newapkbuild does by default and the overwhelming majority does as well.

Reply:
Then I'm not sure why this is considered a blocking issue.
"should" generally implies a stronger reason than "this is common".

Reply:
It isn't blocking but is a deviation from a **super heavily** established norm

Reply:
I don't see how a change from `'` to `"` or vice versa is a significant deviation, or a significant norm.
On the other hand, `'` is less error prone (since it explicitly outlines constants, and prevents accidental escapes).
This honestly looks like bikeshedding over ultimately inconsequential details.
Since the practice is bound to cause significant time wastage in the future, I would rather settle this definitively in a preemptive manner.

Should we perhaps establish a policy regarding quoting rules?
(as another possible target, `"$srcdir"/something` vs `"$srcdir/something"` vs `$srcdir/something` and so on)

Reply:
> Should we perhaps establish a policy regarding quoting rules?

Do what you will just write it somewhere official so i can implement on `atools`.

I personally prefer keeping double quotes because they (subjectively) are easier to distinguish at than single quote and is used by default and nearly all the time. Overall i don't care about what is decided i just want it done consistently **everywhere**.